tensorflow/compiler/xla/tests/build_defs.bzl

```diff
@@ -227,15 +231,18 @@ def generate_backend_test_macros(backends=[]):
   if not backends:
     backends = all_backends
   for backend in filter_backends(backends):
+    this_copts = ["-DXLA_PLATFORM=\\\"%s\\\"" % backend.upper()]
+    if backend in plugins:
+      this_copts += plugins[backend]["test_macro_copts"]
```

Comment:
I would be tempted to leave the -DXLA_DISABLED_MANIFEST stuff in this file and have the plugin provide only the filename.